apps/editing-toolkit/editing-toolkit-plugin/coming-soon/coming-soon.php

```diff
@@ -33,13 +30,22 @@ function show_coming_soon_page() {
  * Renders a fallback coming soon page
  */
 function render_fallback_coming_soon_page() {
+	if ( ! defined( 'GRAVATAR_HOVERCARDS__DISABLE' ) ) {
+		define( 'GRAVATAR_HOVERCARDS__DISABLE', true );
+	}
+
+	add_filter( 'wpcom_disable_logged_out_follow', '__return_true', 10, 1 ); // Disable follow actionbar.
+	add_filter( 'wpl_is_enabled_sitewide', '__return_false', 10, 1 ); // Disable likes.
+	// Disable WP scripts, social og meta, cookie banner.
 	remove_action( 'wp_enqueue_scripts', 'wpcom_actionbar_enqueue_scripts', 101 );
 	remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
 	remove_action( 'wp_print_styles', 'print_emoji_styles' );
 	remove_action( 'wp_head', 'header_js', 5 );
 	remove_action( 'wp_head', 'global_css', 5 );
 	remove_action( 'wp_footer', 'wpcom_subs_js' );
 	remove_action( 'wp_footer', 'stats_footer', 101 );
+	add_filter( 'jetpack_disable_eu_cookie_law_widget', '__return_true', 1 );
+	add_filter( 'jetpack_enable_opengraph', '__return_false', 1 );
 	wp_enqueue_style( 'recoleta-font', '//s1.wp.com/i/fonts/recoleta/css/400.min.css', array(), PLUGIN_VERSION );
```

Comment:
Maybe a blank line before the style here?

Reply:
On it :)